csharp/device/Microsoft.Azure.Devices.Client/DeviceClient.cs

```diff
@@ -92,12 +98,7 @@ public static DeviceClient Create(string hostname, IAuthenticationMethod authent
         /// <returns>DeviceClient</returns>
         public static DeviceClient CreateFromConnectionString(string connectionString)
         {
-#if WINDOWS_UWP
-            return CreateFromConnectionString(connectionString, TransportType.Http1);
```

Comment:
A bad merge.
I've commit the correction to these constructors to match the code in develop branch.
Note that the default being AMQP won't work because that is still not implemented.